primitives/src/lib.rs

```diff
@@ -436,7 +438,7 @@ pub fn retrieve_gateway_pointers(gateway_id: ChainId) -> Result<Vec<GatewayPoint
     Ok(vec![GatewayPointer {
         id: gateway_id,
         gateway_type: GatewayType::ProgrammableExternal(0),
-        vendor: GatewayVendor::PolkadotLike,
+        vendor: GatewayVendor::Rococo,
```

Comment:
Not a fun of introducing separate Rococo as a Vendor. was supposed to be PolkadotLike. GatewayVendor was used to select a correct confirmation plugin to decode ingress messages. 

Since now we have 3 types of decoders: XBI vs PolkadotLike (GrandpaFV) vs EVMBased I suggest we leave Vendor as it was and use PolkadotLike (or SubstrateBased u can rename) for traffic from Rococo or Kusama

Reply:
The decoders are moved out of protocol and are now in the FV, so we don't need the vendor for the confirmation plug anymore (https://github.com/t3rn/protocol/blob/portal_3/src/side_effects/confirm/protocol.rs#L12). 

From the portal side I don't need a mapping for something like EVM vs Substrate, I need a mapping for consensus system (roco, ksma, pdot, etc.). So for me this works perfectly, but I dont know what you need for XBI

Reply:
`fn do_xbi_exit` in Circuit handles it, so for now GV::InternalXBI is unused, feel free to remove 👋

Reply:
Ok. We can always find the correct FV with ChainId via xdns, which I assume will also work for XBI

Reply:
I have updated the GatewayVendors and added Polkadot, Kusama and Ethereum. (also because otherwise new enum order messes with e2e tests)